src/gcp_scanner/crawl.py

```diff
@@ -341,16 +341,15 @@ def get_bucket_names(project_name: str, credentials: Credentials,
                      dump_fd: io.TextIOWrapper
                      ) -> Dict[str, Tuple[Any, List[Any]]]:
   """Retrieve a list of buckets available in the project.
-
   Args:
     project_name: A name of a project to query info about.
     credentials: An google.oauth2.credentials.Credentials object.
     dump_fd: If set, the function will enumerate files stored in buckets and
       save them in a file corresponding to provided file descriptor.
       This is a very slow, noisy operation and should be used with caution.
-
   Returns:
-    A dictionary where key is bucket name and value is a bucket Object.
+    A dictionary where key is bucket name and value is a tuple containing
```

Comment:
"A dictionary with the bucket name as key containing a tuple of the bucket object and a list of root directory objects."